pom.xml

```diff
@@ -768,16 +769,6 @@
             </build>
         </profile>
 
-        <profile>
-            <id>pr-builder</id>
-            <properties>
-                <assembly.skipAssembly>true</assembly.skipAssembly>
-                <!-- Set the classifier to empty in this profile to use regular jar-->
-                <fat.dependency.classifier/>
-                <shade.jar-with-dependencies.phase>none</shade.jar-with-dependencies.phase>
-            </properties>
-        </profile>
-
```

Comment:
Let's not forget to put this back.

Reply:
It's needed to run `DistributionIT`, which we agree to move to the to-be-created `hazelcast-it` module. Should I remove this test to merge the PR or wait for `hazelcast-it`?